packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -54,7 +54,7 @@
     "js-cookie": "2.2.1"
   },
   "devDependencies": {
-    "@justeat/f-button": "1.9.0",
+    "@justeat/f-button": "1.10.1",
```

Comment:
There aren't any changelog entries for some of the package updates, should there be?

Reply:
I haven't updated changelogs for packages where the last update was already "Updated f-button" under "Latest (to be added to the next version)" like [this](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-cookie-banner/CHANGELOG.md#latest-to-be-added-to-next-version)